test/object-store/benchmarks/main.cpp

```diff
@@ -27,6 +27,7 @@
 #pragma comment(lib, "Shlwapi.lib")
 #else
 #include <libgen.h>
+#include <unistd.h>
```

Comment:
extra include here?

Reply:
This is actually needed for `chdir` below. Nothing in CI is compiling these benchmarks, so this has been broken since the catch2 update. The old catch2 headers used to pull it in but not anymore.

Reply:
Can we compile these as part of running the benchmarks on evergreen? Just so we don't have bit-rotting code here?

Reply:
good idea; updated